docs/package.json

```diff
@@ -80,6 +80,7 @@
     "docsearch.js": "^2.6.3",
     "doctrine": "^3.0.0",
     "express": "^4.18.1",
+    "exceljs": "^4.3.0",
```

Comment:
What we do with this peer dependency is strange. It's set as optional but the page broke without, something is off.

Reply:
That's weird, `getExcelJs` isn't called, and therefore `exceljs` shouldn't be imported.
Maybe nextjs does static analysis to check all imports for missing dependencies?

Reply:
I would assume that `const getExcelJs = () => import('exceljs');` will mean that webpack will create a bundle for the import, so that it can be lazy loaded on the client side.

Reply:
Looks like we won't be able to make `exceljs` optional, since webpack checks all imports at compile time and it will always fail if module is not installed.

Reply:
We should consider dependency injection for `exceljs` instead of importing it